src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index];
```

Comment:
Calling `Color.values()` inside a loop or a frequently called method can be inefficient because it creates a new array each time. Consider assigning it to a constant field and using that field here.